packages/app/src/app/overmind/actions.ts

```diff
@@ -224,3 +224,21 @@ export const refetchSandboxInfo: AsyncAction = async ({
     await actions.editor.internal.initializeLiveSandbox(sandbox);
   }
 };
+
+export const acceptTeamInvitation: Action<{ teamName: string }> = (
+  { effects },
+  { teamName }
+) => {
+  effects.analytics.track('Team - Invitation Accepted', {});
```

Comment:
@christianalfoni I could use `actions.track` instead? 🤔

Reply:
This `track` thing is all over the place currently 😂 It is exposed as an action, it is imported directly into components and it is an effect.

We ideally want this to only be used as an effect. That way we later get the freedom to do things like `withTrack('trackThis', myActionDefinition)`. So kinda like how you would use HOC in components to give functionality, the same is possible with actions in Overmind. 

So yeah, go for effect and then we try to clean up the other stuff :)

Reply:
Then this one's ready to be merged 🙂